src/current/cockroachcloud/troubleshooting-page.md

```diff
@@ -162,6 +162,14 @@ Using `sslmode=require` can leave your cluster vulnerable to MITM and impersonat
 
 ## CockroachDB {{ site.data.products.serverless }}
 
+### Delayed cluster access
+
+To enhance security, CockroachDB {{ site.data.products.serverless }} uses authentication throttling tracked per proxy and per (client IP, serverless cluster) pair. This means if multiple login attempts fail (due to wrong passwords or brute force attacks), access is temporarily delayed, with the wait time increasing after each attempt (up to an hour).
+
+<h4>Solution</h4>
+
+If you are experiencing persistent access issues, ensure password configurations are correct or [contact support](https://support.cockroachlabs.com/).
```

Comment:
I guess I'm not sure what this means. I think an admin could reset their password for them -- maybe we should just say contact a cluster administrator or Cockroach Labs support?

Reply:
Wrong passwords can trigger auth throttling. Is that unclear from the text?

Reply:
"ensure password configurations are correct" I guess I'm not sure what this means.

Reply:
Reworded it to say "ensure that the password is correct"